src/views/Home.vue

```diff
@@ -1,23 +1,82 @@
 <template>
   <div>
-    <GrantsOfficeHome v-if="role === 'admin'" />
-    <AgencyHome v-if="role === 'reporter'" />
+    <div class="row m-3 p-3 border border-success rounded" v-if="viewingOpenPeriod">
+      <div class="col" v-if="isAdmin">
+        <a :href="downloadUrl()" class="btn btn-primary">Download Treasury Report</a>
+      </div>
+
+      <div class="col" v-if="isAdmin && false">
+        <a href="/api/audit_report" class="btn btn-info">Download Audit Report</a>
+      </div>
+
+      <div class="col">
+        <button @click.prevent="startUpload" class="btn btn-primary">Submit Spreadsheet</button>
+      </div>
```

Comment:
```suggestion
      <div class="col" v-if="isAdmin">
        <a :href="downloadUrl()" class="btn btn-primary btn-block">Download Treasury Report</a>
      </div>

      <div class="col" v-if="isAdmin && false">
        <a href="/api/audit_report" class="btn btn-info btn-block">Download Audit Report</a>
      </div>

      <div class="col">
        <button @click.prevent="startUpload" class="btn btn-primary btn-block">Submit Spreadsheet</button>
      </div>
```

This is just a suggestion.  See screenshot above.